package.json

```diff
@@ -68,7 +68,7 @@
         "description": "add/remove/create CLI plug-ins"
       }
     },
-    "commands": "./lib/commands",
+    "commands": "./lib/sfdxcommands",
```

Comment:
@rbergman this will make it so we don't have the hidden oclif commands we don't want on suggestions.

Reply:
nice!